src/main/webapp/components/index-cards/index-cards.js

```diff
@@ -107,14 +108,15 @@ export const IndexCardItem = props => {
           title={title}
           subheader={subHeader || moment(modified).fromNow()}
           action={headerAction}
+          titleTypographyProps={titleTypographyProps || ''}
```

Comment:
![image](https://user-images.githubusercontent.com/36056781/78507108-ffafc180-7732-11ea-8b07-799307671ff6.png)

Currently the result index cards header makes the result index cards a little big. This will let us customize it if we ever wanted to change how the result cards look.